README.md

```diff
@@ -1,7 +1,19 @@
 # Info Frontend
 
-This application exposes the user needs and metrics about content on GOV.UK.
+This application exposes the user needs and metrics about content on GOV.UK. It does this by fetching content details (including user needs) from [Content Store](https://github.com/alphagov/content-store).
 
-It fetches content details (including user needs) from the [`content-store`](https://github.com/alphagov/content-store).
+Information pages have a prefix of `/info`, for example: https://www.gov.uk/info/benefits-calculators and https://www.gov.uk/info/employers-sick-pay
```

Comment:
Minor: we could shorten these to `/benefits-calculators` and `employers-sick-pay` links.

Reply:
Hmm, I like them as full links as I think it's clearer and the person reading doesn't have to click through to the page to get the gist of the comment.